app/models/place.rb

```diff
@@ -29,4 +33,52 @@ def self.prefectures_with_key
       }
     end
   end
+
+  def self.create_index!(options={})
+    client = __elasticsearch__.client
+    settings index: {
+               analysis: {
+                 tokenizer: {
+                   ngram_tokenizer: {
+                     type: 'nGram',
+                     min_gram: '2',
+                     max_gram: '3',
+                     token_chars: ['letter', 'digit']
+                   }
+                 },                 
+                 analyzer: {
+                   ngram_analyzer: {
+                     tokenizer: "ngram_tokenizer"
+                   },
+                   kuromoji_analyzer: {
+                     tokenizer: 'kuromoji_tokenizer',
+                     filter:    ['kuromoji_baseform'],
+                   }
+                 }
+               }
+             } do
+      mapping _source: { enabled: true },
+      properties: { enabled: true, analyzer: 'kuromoji_analyzer' } do
+        indexes :trendword_id, type: :integer, index: 'not_analyzed'
+        indexes :name, type: :string, index: 'analyzed', analyzer: 'kuromoji_analyzer'
+        indexes :location, type: :geo_point, lat_lon: true
+      end
+    end
+    client.indices.delete index: 'places' rescue nil if options[:force]
+    client.indices.create index: 'places',
+      body: {
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.